processing/src/main/java/org/apache/druid/query/QueryInterruptedException.java

```diff
@@ -44,7 +44,7 @@
 public class QueryInterruptedException extends QueryException
 {
   public static final String QUERY_INTERRUPTED = "Query interrupted";
-  public static final String QUERY_CANCELLED = "Query cancelled";
+  public static final String QUERY_CANCELED = "Query cancelled";
```

Comment:
Too bad the "cancelled" spelling is a documented error code. Echoes of the HTTP "Referer" header.

Reply:
I've noticed a number of British spellings in the code. Must have been done by a Brit. Perhaps we can credit this spelling to that person as well. Though we can't change the text, it is safe to change the constant name, right? Added a comment to explain the disparity.

Reply:
Yeah, it's safe to change the constant name.